Networking/Networking/Remote/RefundsRemote.swift

```diff
@@ -35,6 +35,25 @@ public final class RefundsRemote: Remote {
         enqueue(request, mapper: mapper, completion: completion)
     }
 
+    /// Create a refund by `orderID`.
+    ///
+    /// - Parameters:
+    ///     - siteID: Site for which we'll emit a refund.
+    ///     - orderID: Unique identifier for the resource "order" for which you are emitting a refund.
+    ///     - refund: The Refund model used to create the custom entity for the request.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func createRefund(for siteID: Int,
+                                 by orderID: Int,
+                                 refund: Refund,
+                                 completion: @escaping (OrderRefund?, Error?) -> Void) {
+        let path = "\(Path.orders)/" + String(orderID) + "/" + "\(Path.refunds)"
```

Comment:
I'm just curious here, do we really need to interpolate Path.orders and Path.refunds?

Reply:
It's not needed, but I adapted to the style used in all the other Remote classes, like `CommentRemote`, `OrdersRemote` and so on.
The only Remote class that doesn't use this approach is `DevicesRemote`, which construct the path with formatted strings `"devices/%@/delete"`. Anyway, as told in the first PR https://github.com/woocommerce/woocommerce-ios/pull/1175/files/865edea5221c6fde8b264420963b4218a416cb7f#diff-aad32630f4e1512055993b8a5ebf354f, this type of approach will be changed, but I'm open to any proposal :)

Reply:
🤔 I don't like doing things in a particular way just because "that's the way we do it", but on the other hand, there are going to be arguments about consistency...

I don't care too much about consistency for the shake of consistency, but I understand others find it very important. So, leave as it is?

Reply:
I agree with you about the idea of consistency. We must always pursue the best solution. But I think that since this will affects differents classes, probably it's better to do it in a separated PR? I preferred to conform, rather than finding ourselves with 3 different ways of declaring a path.